Cesium3DTilesSelection/src/TilesetLoadTilesetDotJson.cpp

```diff
@@ -261,16 +278,17 @@ LoadResult Tileset::LoadTilesetDotJson::Private::workerThreadHandleResponse(
         *pRootTile,
         tileset,
         *pContext,
+        asyncSystem,
```

Comment:
workerThreadLoadTerrainTile() now does spawn a new task, so it can't use the Tile, Tileset, and TileContext by reference as there are more than two threads access them both at the same time